src/main/resources/assets/neoforge/lang/en_us.json

```diff
@@ -119,6 +119,14 @@
   "commands.neoforge.timespeed.query.default": "Time in %s flows normally (20 minutes per day).",
   "commands.neoforge.timespeed.set": "Set flow of time in %s to %sx (%s minutes per day).",
   "commands.neoforge.timespeed.set.default": "Set flow of time in %s to default (20 minutes per day).",
+  "commands.neoforge.data_components.list.error.held_stack_empty": "Held stack must not be empty",
+  "commands.neoforge.data_components.list.title": "Data components on %s:",
+  "commands.neoforge.data_components.list.entry": "\n  - %s",
```

Comment:
I must say (only to note it), I'm not particularly a fan of using `\n` in translation keys, but that's an item to tackle for the future.